code/modules/holomap/poi.dm

```diff
@@ -0,0 +1,18 @@
+
+/obj/effect/landmark/minimap_poi
+	name = "minimap poi"
+	desc = null
+	icon = 'icons/effects/map_effects.dmi'
+	icon_state = "minimap_poi"
+
+/obj/effect/landmark/minimap_poi/Initialize()
+	..()
+	return INITIALIZE_HINT_LATELOAD
+
+/obj/effect/landmark/minimap_poi/LateInitialize()
+	. = ..()
+	SSholomap.pois += src
+
+/obj/effect/landmark/minimap_poi/Destroy()
+	. = ..()
+	SSholomap.pois -= src
```

Comment:
Clean up after yourself first, unless there's a compelling reason not to, in Destroy()

Reply:
done

Reply:
not sure I understand - do you mean to  mpve the `-= src` above the `. = ..()`?

Reply:
Yes

Reply:
done